staging/src/k8s.io/code-generator/cmd/validation-gen/validation.go

```diff
@@ -533,6 +505,90 @@ func (td *typeDiscoverer) discoverStruct(thisNode *typeNode, fldPath *field.Path
 // are being processed by this generator. For cross-package calls we will need
 // to verify the target, either by naming convention + fingerprint or by
 // explicit comment-tags or something.
+// discoverAlias walks an alias type recursively.
```

Comment:
Looks like godoc for getValidationFunctionName got misplaced here.

Reply:
oops, merge issue